src/custom/hooks/useDebounceWithForceUpdate.ts

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react'
+
+// modified from https://usehooks.com/useDebounce/
+export default function useDebounceWithForceUpdate<T>(
+  value: T,
+  delay: number,
+  forceUpdateRef?: string | number | boolean
```

Comment:
a bit weird signature. Probably this can be `any` type, since it's what useEffects accepts

why `Boolean(forceUpdateRef)`?  this is a general hook. If `forceUpdateRef` chanfges from `'something'` to `''` then you wipn't force the update, right?